packages/members-api/lib/repositories/product.js

```diff
@@ -133,14 +134,15 @@ class ProductRepository {
         }
 
         const productData = {
+            type: 'paid',
             name: data.name,
             description: data.description,
             benefits: data.benefits
         };
 
         const product = await this._Product.add(productData, options);
 
-        if (this._stripeAPIService.configured) {
+        if (this._stripeAPIService.configured && data.type !== 'free') {
```

Comment:
Ignores any stripe changes for `free` tier

Reply:
This isn't necessary as you cannot create free tiers, right?

Reply:
Ah yea good point, we don't need that for adding new tiers.